src/app/timer/timer.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+import { interval, subscribeOn, Subscription, timer, fromEvent, from } from 'rxjs';
+
+
+@Component({
+  selector: 'app-timer',
+  templateUrl: './timer.component.html',
+  styleUrls: ['./timer.component.css'],
+})
+export class TimerComponent implements OnInit {
```

Comment:
structure of component class:
- public variables
- private variables
- constructor (if exists)
- lifecycle hooks (if any)
- public methods
- private methods

please align appropriately